modules/bound-sa-tokens-configuring.adoc

```diff
@@ -67,13 +67,32 @@ If the output shows a message similar to one of the following messages, the upda
 ** `3 nodes are at revision 11; 0 nodes have achieved new revision 12`
 ** `2 nodes are at revision 11; 1 nodes are at revision 12`
 
-... Manually restart all pods in the cluster:
+.. Optional: Force the holder to request a new bound token either by performing a rolling node restart or by manually restarting all pods in the cluster.
++
+// TODO: When would a user NOT need to do this?
+
+*** Perform a rolling node restart:
++
+[WARNING]
+====
+If you have workloads running on your cluster, performing a rolling node restart can cause a service interruption. TODO: What to say about how to "be careful" to minimize disruption?
```

Comment:
@stlaz @mfojtik Let me know if you have any suggestions for this warning wording for rolling node restarts if you have workloads. I really wasn't sure what to tell them to do, on how to be careful to minimize disruption. Thoughts?

Reply:
No way to mitigate disruptions. I would avoid doing this when there are custom workloads running in the cluster and would only propose to do this with the default cluster as installed.

Reply:
@stlaz So would you recommend that we say something like this?

```
It is not recommended to perform a rolling node restart if you have custom workloads running on your cluster, because it can cause a service interruption. Instead, manually restart all pods in the cluster.
```

I guess what seems odd about adding this node restart option is - how many people are needing to actually do this procedure on a fresh cluster? Wouldn't most people be coming into this procedure already having workloads running in their cluster?